src/planner/builder/builder_test.go

```diff
@@ -338,6 +332,26 @@ func TestProcessSelect(t *testing.T) {
 					Range:   "[512-4096)",
 				}},
 		},
+		{
```

Comment:
The test to the old master branch is still successful.
Please add the test to the  old master branch is failed. but to the pr is successful.